artemis-docker/Dockerfile-centos7-19

```diff
@@ -17,7 +17,7 @@
 
 # ActiveMQ Artemis
 
-FROM eclipse-temurin:11-centos7
+FROM eclipse-temurin:19-jdk-centos7
```

Comment:
This change probably doesnt make sense. Java 19 isnt really supported anymore, so similarly the Temurin Java 19 based images [are not either](https://github.com/docker-library/docs/blob/master/eclipse-temurin/README.md#supported-tags-and-respective-dockerfile-links). I'd either leave it at 11 until centos7 itself soon isnt (EOL in June) or use the Java 17 image variant, since both those JDKs/images will already be more up to date in terms of Java security fixes than the Java 19 one ever will be.

Reply:
Fixed.